pkg/sql/plan/base_binder.go

```diff
@@ -1219,3 +1226,25 @@ func resetDateFunction(dateExpr *Expr, intervalExpr *Expr) ([]*Expr, error) {
 	}
 	return resetDateFunctionArgs(dateExpr, listExpr)
 }
+
+func resetTimeStampUnitFunc(unit *Expr) (*Expr, error) {
+	intervalTypeStr := unit.Expr.(*plan.Expr_C).C.Value.(*plan.Const_Sval).Sval
+	intervalType, err := types.IntervalTypeOf(intervalTypeStr)
+	if err != nil {
+		return nil, err
+	}
+	intervalTypeInFunction := &plan.Type{
+		Id:   int32(types.T_int64),
+		Size: 8,
+	}
+	return &plan.Expr{
```

Comment:
you can use makePlan2Int64ConstExprWithType

Reply:
ok

Reply:
this file can't import "github.com/matrixorigin/matrixone/pkg/sql/plan"(because of cycle import)， so we can‘t use makePlan2Int64ConstExprWithType